src/components/ComponentPages/NickName/NickNameUI/index.tsx

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import { Table, Input, Select, Form, Typography, Button, Modal } from "antd";
+import Icon, { PlusCircleOutlined } from "@ant-design/icons";
+import styles from "./NickName.module.scss";
+import messages from "../../../../messages";
+
+const originData = [];
+const { Option } = Select;
+
+for (let i = 0; i < 5; i++) {
```

Comment:
Not sure why we use `5`

Reply:
This is for dummy data as API is not integrated its for front end design